Documentation/Reference/Columns/Check/Index.rst

```diff
@@ -159,6 +159,8 @@ showIfRTE
          If set, this field will show *only* if the RTE editor is enabled
          (which includes correct browser version and user-rights altogether.)
 
+         This property is deprecated since TYPO3 v8.
```

Comment:
please just remove: the option was removed and we don't keep the deprecation notices within the docs anymore.